pkgs/misc/emulators/openmsx/default.nix

```diff
@@ -6,16 +6,17 @@
 , freetype, tcl, zlib
 }:
 
+with stdenv.lib;
```

Comment:
This seems to be redundant. Is this intended?

Reply:
Why would it be redundant? Not using it as an argument, but I always imported `stdenv.lib` in my scripts.

Reply:
You did not add anything that uses `stdenv.lib` in the commit and the expression must have been evaluating before you added this line. I'm just asking in case you intended something else.

Reply:
Ah, yes. I intended to translate the lines, but I forgot to eliminate one of them.